CESC-submission/paper.tex

```diff
@@ -62,15 +62,15 @@
+Escalating demand for digital services has motivated building non-traditional solutions to reduce costs and achieve a greater level of flexibility. Among these solutions, peer-assisted models are gaining more popularity in replacing infrastructure-based (possibly) centralized approaches. Cryptocurrencies have strengthened this trend by providing a fully distributed mechanism to compensate service provision, and their blockchains and consensus protocols offer a trustless and publicly verifiable way to govern the system. We present a generic framework for building distributed service markets by utilizing these new technologies. We also discuss the security and efficiency challenges, with a focus on how the economic considerations impact system design and threat mitigation, and conclude with some potential solutions.
 \end{abstract}
 
 
 \section{Introduction}
-Usually, when obtaining digital services, we deal with traditional systems that are centrally managed. Most of the time, we resort to third party providers, e.g., commercial companies,  to obtain services like file storage, content distribution, computation outsourcing, and many others. Despite being effective and widely deployed, this centrally-managed paradigm introduces several trust, cost, and transparency issues. It requires establishing complex business relationships with these companies, where customers usually overprovision their needs in order to handle future peak demands~\cite{Kassa13,Nicolae14}. Also, it constrains the customers with the service specifications these companies can offer, like the geographic coverage and service speed, not to mention the limited visibility into the real status of the system regarding its performance and the available amount of resources.
+Usually, when obtaining digital services, we deal with traditional systems that are centrally managed. Most of the time, we resort to third party providers, e.g. commercial companies, to obtain services like file storage, content distribution, outsourced computation, and many others. Despite being effective and widely deployed, this centrally-managed paradigm introduces several trust, cost, and transparency issues. It requires establishing complex business relationships with these companies, where customers usually overprovision their needs in order to handle future peak demands~\cite{Kassa13,Nicolae14}. Also, it constrains the customers with the service specifications these companies can offer, such as geographic coverage and service speed, not to mention the limited visibility into the real status of the system regarding its performance and the available resources.
```

Comment:
computation outsourcing -> outsourced computation
like the geographic -> such as geographic